dataplane/linux/bpf_ep_mgr.go

```diff
@@ -749,6 +752,7 @@ func (m *bpfEndpointManager) attachWorkloadProgram(ifaceName string, endpoint *p
 	//   when we cannot encap the packet - non-GSO & too close to veth MTU
 	ap.TunnelMTU = uint16(m.vxlanMTU - 50)
 	ap.IntfIP = calicoRouterIP
+	ap.ToHostMark = uint32(m.bpfToHostMark)
```

Comment:
I think marking all traffic from a workload will be a problem.  For secondary ENI pods, workload traffic that is _not_ responding to a  nodeport _should_ go over the secondary ENI but I think this mark will force it over the primary ENI.